pkg/common/vclib/datacenter_test.go

```diff
@@ -133,7 +133,7 @@ func TestDatacenter(t *testing.T) {
 		t.Error(err)
 	}
 
-	diskPath := ds.Path(avm.Name + "/disk1.vmdk")
+	diskPath := (ds.Datastore).Path(avm.Name + "/disk1.vmdk")
```

Comment:
cosmetic: no need for parens there.

Reply:
Yup... was left over from my debugging a while back